src/sql/sqlops.proposed.d.ts

```diff
@@ -951,6 +951,12 @@ declare module 'sqlops' {
 				 */
 				message: DialogMessage;
 
+				/**
+				 * Set the dialog name when opening
+				 * the dialog for telemetry
+				 */
+				dialogName: string;
```

Comment:
should this be optional?  i.e., is this going to be a breaking change for anyone?